platform/mellanox/smartswitch/dpuctl/dpuctl.sh

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env bash
+#
+# Copyright (c) 2024 NVIDIA CORPORATION & AFFILIATES.
+# Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# By default the dark mode is enabled
+DARK_MODE=true
+
+bf3_pci_id="15b3:c2d5"
+dpu2pcie[0]="06:00.0"
```

Comment:
Is this static BDF? Will it be same for all HWSKU's of smartswitch?

Reply:
Yes, the mapping will be the same.